docs(pie-docs): DSW-1890 add motion chapter on docsite (#1474)
justeattakeaway/pie: apps/pie-docs/src/foundations/motion/overview/overview.md

The objectives of these guidelines is to provide guidance for using motion in our UI elements, and to empower designers to design with motion in mind so that we can evoke an emotive experience.

---

## Principles

These principles serve as a guide when using motion in our products.

{% mediaElementList {
    data: principlesData
} %}

---

## How we use motion

#### Do’s & Don’ts

{% usage {
do: {
  type: usageTypes.list,
  items: [
    "Apply motion effects that are contextually relevant to the task at hand.",
    "Ensure that motion effects do not slow down the user or hinder task completion.",
    "Keep animations and transitions at an appropriate speed to maintain a sense of efficiency.",
    "Ensure easing is always used to create smooth and natural motion."
  ]
},
dont: {
  type: usageTypes.list,
  items: [
    "Use motion for the sake of it, motion should always have a clear purpose or intent and contribute to the user experience.",
    "Use motion excessively, avoid overwhelming users with excessive or distracting motion.",
    "Use motion inconsistently, which can confuse users and reduce predictability."
  ]
}
} %}

---

## Accessibility

> We must also provide options for transitions without animation that retain it’s context and meaning to the user to that reduced motion can be applied for users that are sensitive or adverse to motion.

Reviewer: ```suggestion
We must also provide options for transitions without animation that retain context and meaning to the user so that reduced motion can be applied for users that are sensitive or averse to motion.
```
